src/main.ts

```diff
@@ -2,13 +2,24 @@ import { NestFactory } from '@nestjs/core';
 import { NestExpressApplication } from '@nestjs/platform-express';
 import { AppModule } from './app.module';
 import { join } from 'path';
+import { handlebars } from 'hbs';
 
 async function bootstrap() {
   const app = await NestFactory.create<NestExpressApplication>(AppModule);
   app.useStaticAssets(join(__dirname, '..', 'public'));
   app.setBaseViewsDir(join(__dirname, '..', 'views'));
+  registerHandlebarsHelpers();
   app.setViewEngine('hbs');
   await app.listen(3000);
 }
 
+export function registerHandlebarsHelpers() {
+  handlebars.registerHelper('ifCond', function (v1, v2, options) {
+    if (v1 === v2) {
+      return options.fn(this);
+    }
+    return options.inverse(this);
+  });
+}
```

Comment:
It could be extracted into a handlebars helper module because i think it will soon be used into your e2e tests too !